core/domain/user_services.py

```diff
@@ -179,6 +181,8 @@ def validate(self):
         if not is_user_id_correct(self.user_id):
             raise utils.ValidationError('The user ID is in a wrong format.')
 
+        # TODO(#10178): Remove check for gae_id attribute below
+        # when feconf.ENABLE_USER_AUTH_MODEL flag has been set True.
```

Comment:
You have already put a TODO on top of this class so you do not need to repeat them (here and elsewhere in the other files). You can put a TODO at the storage model as well (i.e. to make required = false when the flag is enabled)